### DISCUSSION TITLE
Please revert our repository that caused by GitHub DMCA takedown

### DISCUSSION THREAD

**Discussion by itsgitz** [community/community#49033] [Other Feature Feedback, Questions, & Ideas] [answered]
We couldn't access our repository (https://github.com/Zhupy17/Sinarkencana) that contains our Odoo custom modules caused by GitHub DMCA takedown https://github.com/github/dmca/blob/master/2023/02/2023-02-22-droggol-themes-2.md. We already send an email to Droggol support, and they said I should create a ticket or start a discussion on this issue. Honestly, we have paid for this module https://github.com/Zhupy17/Sinarkencana/tree/main/.data/backup/droggol_theme_common on our Odoo system. Now, I have no idea what should I do. We couldn't perform any actions because the entire modules from our repository is gone. Any suggestions?

Thank you

**Comment by queenofcorgis** [comment, marked as answer]
Hi @itsgitz! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The community is unable to assist in matters like this. Please open a ticket with them :)
